README.md

```diff
@@ -1,35 +1,80 @@
-[![License: APGL](https://img.shields.io/github/license/telefonicaid/perseo-fe.svg)](https://opensource.org/licenses/AGPL-3.0)
-<br/>
-[![Build badge](https://img.shields.io/travis/telefonicaid/perseo-fe.svg)](https://travis-ci.org/telefonicaid/perseo-fe/)
-[![Coverage Status](https://coveralls.io/repos/github/telefonicaid/perseo-fe/badge.svg?branch=master)](https://coveralls.io/github/telefonicaid/perseo-fe?branch=master)
+[![License: AGPLv3+](https://img.shields.io/badge/License-AGPLv3+-blue.svg)](./LICENSE)
+[![Docker Status](https://img.shields.io/docker/pulls/fiware/perseo.svg)](https://hub.docker.com/r/fiware/perseo/)
+[![Support badge](https://img.shields.io/badge/tag-fiware--perseo-orange.svg?logo=stackoverflow)](https://stackoverflow.com/questions/tagged/fiware-perseo)
+<br> [![Documentation badge](https://img.shields.io/readthedocs/perseo.svg)](https://perseo.readthedocs.io/en/latest/)
+[![Build Status](https://travis-ci.org/telefonicaid/perseo-fe.svg?branch=master)](https://travis-ci.org/telefonicaid/perseo-fe)
 ![Status](https://nexus.lab.fiware.org/static/badges/statuses/perseo.svg)
 
-CEP implementation for IoT platform
+## Overview
 
-## Index
+Perseo is a Complex Event Processing (CEP) software designed to be fully _NGSIv2_-compliant. It uses NGSIv2 as the
+communication protocol for events, and thus, Perseo is able to seamless and jointly work with _context brokers_ such as
+[Orion Context Broker](https://github.com/telefonicaid/fiware-orion).
 
--   [Architecture](documentation/architecture.md)
--   [Logs & Alarms](documentation/logs.md)
--   O&M
+This project is part of [FIWARE](https://www.fiware.org). You can find more FIWARE components in the
+[FIWARE catalogue](https://catalogue.fiware.org).
 
-    -   [Deployment](documentation/deployment.md)
-    -   [Configuration](documentation/configuration.md)
-    -   [Administration](documentation/admin.md)
-    -   [Metrics API](documentation/metrics_api.md)
+| :books: [Documentation](https://perseo.readthedocs.io/en/latest) | :whale: [Docker Hub](https://hub.docker.com/r/fiware/perseo/) | :dart: [Roadmap](documentation/roadmap.md) |
+| ---------------------------------------------------------------- | ------------------------------------------------------------- | ------------------------------------------ |
 
--   [API](documentation/api.md)
-    -   [Plain rules](documentation/plain_rules.md)
-    -   [Errors](documentation/errors.md)
--   [Data model](documentation/models.md)
--   [Available actions for PEP](documentation/pep_actions.md)
--   [Documentation for developers of perseo](documentation/development.md)
 
-| :dart: [Roadmap](documentation/roadmap.md) |
-| ------------------------------------------ |
+## Content
 
+-   [Background](#background)
+-   [Installation](#installation)
+-   [Usage](#usage)
+-   [API](#api)
+-   [Testing](#testing)
+-   [Quality Assurance](#quality-assurance)
```

Comment:
This entry in the index should be removed.

Reply:
Fixed in 66fb1f8a911267ba09fc840eca020310d554bc2e